app/kuma-dp/cmd/run.go

```diff
@@ -106,5 +106,6 @@ func newRunCmd() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&cfg.ControlPlane.BootstrapServer.URL, "cp-address", cfg.ControlPlane.BootstrapServer.URL, "Mesh that Dataplane belongs to")
 	cmd.PersistentFlags().StringVar(&cfg.DataplaneRuntime.BinaryPath, "binary-path", cfg.DataplaneRuntime.BinaryPath, "Binary path of Envoy executable")
 	cmd.PersistentFlags().StringVar(&cfg.DataplaneRuntime.ConfigDir, "config-dir", cfg.DataplaneRuntime.ConfigDir, "Directory in which Envoy config will be generated")
+	cmd.PersistentFlags().StringVar(&cfg.DataplaneRuntime.TokenPath, "dataplane-token", cfg.DataplaneRuntime.TokenPath, "Path to file in which generated token is stored")
```

Comment:
Is `Path to file in which generated token (via kumactl or HTTP API) is stored` better?

It would be helpful to leave suggestions with comments like this, thanks!

Reply:
E.g., `Path to a file with dataplane token (use 'kumactl generate dataplane-token' to get one)`

Reply:
Changed